YANG/tapi-common.yang

```diff
@@ -144,10 +144,32 @@ module tapi-common {
                 uses lifecycle-state-pac-g;
                 description "none";
             }
+            container capacity {

```

Comment:
CapacityPac (containing total & available Capacity) is already added to the SIP/CSEP. It has just been renamed (dropped the word 'Transfer') since it is now being applied to "points".